api/RTCRtpTransceiver.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpTransceiver",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": "72"
```

Comment:
Transceiver itself landed in 69 https://storage.googleapis.com/chromium-find-releases-static/842.html#842893e6fdc0d81da782035ae314433d5c91df20